code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1432,6 +1560,31 @@ datum
 							for(var/j = 1, j <= rand(1, 3), j++)
 								step(B, pick(NORTH,SOUTH,EAST,WEST))
 
+		slimematerials
+			name = "Slime Materials"
+			id = "m_mats"
+			result = null
+			required_reagents = list("carbon" = 5)
+			result_amount = 1
+			required_container = /obj/item/slime_extract/silver
+			required_other = 1
+			on_reaction(var/datum/reagents/holder)
+				var/obj/item/stack/sheet/S1 = pick(new /obj/item/stack/sheet/plasteel,
```

Comment:
On another note: copy pasting to do it twice is not a good idea, using a `for` loop is much cleaner:

``` DM
for(var/x = 1 to 2)
    var/obj/item/stack/sheet/S = ...
    S.amount = 5
    ...
```